tools/state-viewer/src/apply_chain_range.rs

```diff
@@ -126,6 +126,7 @@ fn apply_block_from_range(
     csv_file_mutex: &Mutex<Option<&mut File>>,
     only_contracts: bool,
     use_flat_storage: bool,
+    use_trie_for_free: bool,
```

Comment:
While technically accurate this name doesn't quite convey the message for me :) How about something like `use_flat_storage_from_trie` or similar?

Reply:
Yeah I didn't like the name
I will go for `use_flat_storage_from_trie` if there aren't any better suggestions!